eng/psakefile.ps1

```diff
@@ -248,6 +262,11 @@ function Build-PyQIR([string]$project) {
             exec {
                 docker run --rm $userSpec -v $ioVolume -v $llvmVolume -e LLVM_SYS_110_PREFIX=/usr/lib/llvm -w /io/$project manylinux2014_x86_64_maturin conda run --no-capture-output python -m tox -e test
             }
+            
+            Write-BuildLog "docker run --rm $userSpec -v $ioVolume -v $llvmVolume -e LLVM_SYS_110_PREFIX=/usr/lib/llvm -w /io/$project manylinux2014_x86_64_maturin conda run --no-capture-output cargo test --release -vv -- --nocapture" "command"
+            exec {
+                docker run --rm $userSpec -v $ioVolume -v $llvmVolume -e LLVM_SYS_110_PREFIX=/usr/lib/llvm -w /io/$project manylinux2014_x86_64_maturin conda run --no-capture-output cargo test --release -vv -- --nocapture
+            }
```

Comment:
Logging commands in general is dangerous, so I've been logging them manually.

Reply:
I'm not referring to logging in general, but a way to avoid duplicating the command. Something like this (pseudocode, since I don't know the best way to do it in PowerShell):

```
trace(command) {
    print command as a string
    execute command
}

trace(docker --rm $userSpec ...)
```

Reply:
@samarsha Pushed new Invoke-LoggedCommand. Please take a look.